Sources/Scheduler.swift

```diff
@@ -169,15 +169,14 @@ public final class QueueScheduler: DateSchedulerProtocol {
 	/// - parameters:
 	///   - qos: Dispatch queue's QoS value.
 	///   - name: Name for the queue in the form of reverse domain.
+	///   - targeting: (Optional) The queue on which this scheduler's work is
+	///     targeted
 	@available(OSX 10.10, *)
-	public convenience init(
-		qos: DispatchQoS = .default,
-		name: String = "org.reactivecocoa.ReactiveSwift.QueueScheduler"
-	) {
-		self.init(internalQueue: DispatchQueue(
-			label: name,
-			qos: qos
-		))
+	public convenience init(qos: DispatchQoS = .default, name: String = "org.reactivecocoa.ReactiveSwift.QueueScheduler", targeting targetQueue: DispatchQueue? = nil) {
```

Comment:
It seemed so…out of place. :)

I can restore it though, since that seems to be the preference.